ee_tests/src/specs/quickstart_deployments.spec.ts

```diff
@@ -131,33 +131,29 @@ describe('Creating new quickstart in OSIO', () => {
     await spacePipelinePage.spaceHeader.deploymentsOption.clickWhenReady();
 
     await browser.sleep(5000);
+
+    support.info('Verifying deployments page');
+
     let spaceDeploymentsPage = new SpaceDeploymentsPage();
 
-    /* Verify that app deployed to stage presents: Success icon, blue circle graphic, text = "Output from run = 1 pods" */
+    let deploymentContainers = await spaceDeploymentsPage.getDeploymentCardContainers();
+    expect(deploymentContainers.length).toBe(1, 'number of deployed applications');
 
-    let textStr = await spaceDeploymentsPage.resourceCardByNameAndIndex(spaceName, STAGE).getText();
-    support.info('1 Output from run = ' + textStr);
-    expect(await spaceDeploymentsPage.allResourceCards.count()).toBe(8);
-    expect(await spaceDeploymentsPage.successfulDeployStatusByNameAndIndex(spaceName, STAGE).isPresent()).toBeTruthy();
+    let deploymentContainer = deploymentContainers[0];
+    expect(deploymentContainer.getApplicationName()).toBe(spaceName, 'application name');
 
-    spaceDeploymentsPage.podRunningTextByNameAndIndex(spaceName, STAGE).getText();
-    support.info('2 Output from run = ' + textStr);
+    let deploymentCard = await deploymentContainer.getDeploymentCards();
+    expect(deploymentCard.length).toBe(2, 'number of environments');
 
-    textStr = await spaceDeploymentsPage.successfulDeployStatusByNameAndIndex(spaceName, STAGE).getText();
-    support.info('3 Output from run = ' + textStr);
+    let stagingCard = deploymentCard[0];
+    expect(stagingCard.getStatus()).toBe(DeploymentStatus.OK, 'Stage environment status');
+    expect(stagingCard.getVersion()).toBe('1.0.1', 'Stage environment version');
+    expect(stagingCard.getPodsCount()).toBe(1, 'Stage environment pods');
+    expect(stagingCard.getRunningPodsCount()).toBe('1 Running', 'Stage environment running pods');
 
-    textStr = await spaceDeploymentsPage.resourceUsageCardByIndex(1).getText();
+    let textStr = await spaceDeploymentsPage.resourceUsageCardByIndex(1).getText();
```

Comment:
1. I don't like the `ByIndex` part 
2. Similarly to previous comment, I would omit `Card` which is implementation detail
3. In general, this method doesn't follow naming conventions of other methods

What about something like this `await deploymentsPage.getResourceUsage().<something>(1).getText();`?